core/src/components/slider/slider.spec.ts

```diff
@@ -68,6 +71,7 @@ describe(`Slider basic`, () => {
 		const maxLabelDirective = slider.directives.maxLabelDirective(maxLabel);
 		const unsubscribe = slider.state$.subscribe((newState) => {
 			state = newState;
+			defaultStateValues.slotLabel = newState.slotLabel;
```

Comment:
@fbasso : we need to discuss how we want to test slots. In this use case, I did the same as pagination, assuring the default function provided by the widget is the same as the expected by our tests. 

But I also noticed in select, you `normalize` the state replacing functions by a string `(function)`, hence you don't check them.

Reply:
As discussed, I changed it to use a computed state with some manual modification (in this case, changing the function for a string `'(function')`, and changing the default config to a function to avoid cloning the object all the time.